src/main/scala/com/cognite/sdk/scala/common/package.scala

```diff
@@ -84,4 +85,25 @@ package object common {
 
   implicit val sequenceToCreateTransformer: Transformer[Sequence, SequenceCreate] =
     Transformer.define[Sequence, SequenceCreate].buildTransformer
+
+  implicit val strSeqToLabelsOnUpdateTransformer
+      : Transformer[Option[Seq[String]], Option[LabelsOnUpdate]] =
+    new Transformer[Option[Seq[String]], Option[LabelsOnUpdate]] {
+      override def transform(src: Option[Seq[String]]) = src match {
+        case Some(value: Seq[String]) =>
+          Some(LabelsOnUpdate(add = Some(value.map(CogniteExternalId))))
+        case _ => None
+      }
+    }
+
+  implicit val extIdSeqToLabelsOnUpdateTransformer
+      : Transformer[Option[Seq[CogniteExternalId]], Option[LabelsOnUpdate]] =
+    new Transformer[Option[Seq[CogniteExternalId]], Option[LabelsOnUpdate]] {
+      override def transform(src: Option[Seq[CogniteExternalId]]) = src match {
+        case Some(value: Seq[CogniteExternalId]) => Some(LabelsOnUpdate(add = Some(value)))
+        case _ => None
+      }
+    }
+
+  implicit val labelsOnUpdateEncoder: Encoder[LabelsOnUpdate] = deriveEncoder
```

Comment:
Not sure these should be in package.scala. Is it ok?

Reply:
And maybe, it would be nice to merge these two transformers into one having cases for different source types? If agreed, I could use some help there :D